vendor/modules.txt

```diff
@@ -1217,7 +1217,6 @@ golang.org/x/net/context/ctxhttp
 golang.org/x/net/http/httpguts
 golang.org/x/net/http/httpproxy
 golang.org/x/net/http2
-golang.org/x/net/http2/h2c
```

Comment:
Hhmm so are we not using h2c anymore?

Reply:
We're not. It was being used to upgrade `http/1.1` connections to `http/2`. Now, we're just serving `http/2` here https://github.com/grafana/loki/blob/27b9fe50f0fe85607706a0ac6aa45ad032d0bf0a/pkg/storage/chunk/cache/groupcache.go#L133-L157